src/app/component/Header/Header.container.js

```diff
@@ -45,14 +47,16 @@ export const mapStateToProps = (state) => ({
     logo_alt: state.ConfigReducer.logo_alt,
     isLoading: state.ConfigReducer.isLoading,
     device: state.ConfigReducer.device,
-    activeOverlay: state.OverlayReducer.activeOverlay
+    activeOverlay: state.OverlayReducer.activeOverlay,
+    isWishlistLoading: state.WishlistReducer.isLoading
 });
 
 /** @namespace Component/Header/Container/mapDispatchToProps */
 export const mapDispatchToProps = (dispatch) => ({
     showOverlay: (overlayKey) => dispatch(toggleOverlayByKey(overlayKey)),
     hideActiveOverlay: () => dispatch(hideActiveOverlay()),
     setNavigationState: (stateName) => dispatch(changeNavigationState(TOP_NAVIGATION_TYPE, stateName)),
+    showPopup: (payload) => dispatch(showPopup(SHARE_WISHLIST_POPUP_ID, payload)),
```

Comment:
I think, this name is misleading, this will trigger the wish-list popup, not just any popup.